mmv1/third_party/terraform/services/redis/resource_redis_cluster_test.go.erb

```diff
@@ -0,0 +1,138 @@
+<% autogen_exception -%>
+package redis_test
+<% unless version == "ga" -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+// Validate that replica count is updated for the cluster
+func TestAccRedisCluster_updateReplicaCount(t *testing.T) {
+	t.Parallel()
+
+	name := fmt.Sprintf("tf-test-%d", acctest.RandInt(t))
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckRedisClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				// create cluster with replica count 1
+				Config: createOrUpdateRedisCluster(name /* replicaCount = */, 1 /* shardCount = */, 3, true),
+			},
+			{
+				ResourceName:      "google_redis_cluster.test",
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
Because psc_configs is marked as ignore_read we have to tell the import step of the test to ignore comparison on that field. During import we don't have the original config to grab the initial value for psc_configs from, and we don't get the correct value back from the API so we end up with an empty psc_configs.

You can add something like:
`ImportStateVerifyIgnore: []string{"psc_configs"},` 
to each ImportState step in both of the tests in this file

Reply:
Thanks for the suggestion. Done and this helped the tests pass locally.